great_expectations/datasource/fluent/spark_file_path_datasource.py

```diff
@@ -36,9 +36,36 @@ def _get_reader_options_include(self) -> set[str] | None:
         return {"header", "infer_schema"}
 
 
+class ParquetAsset(_FilePathDataAsset):
+    type: Literal["parquet"] = "parquet"
+    # The options below are available for parquet as of spark v3.4.0
+    # See https://spark.apache.org/docs/latest/sql-data-sources-parquet.html for more info.
+    datetime_rebase_mode: Literal["EXCEPTION", "CORRECTED", "LEGACY"] = Field(alias="datetimeRebaseMode")
+    int_96_rebase_mode: Literal["EXCEPTION", "CORRECTED", "LEGACY"] = Field(alias="int96RebaseMode")
+    merge_schema: bool = Field(False, alias="mergeSchema")
+
+    class Config:
+        extra = pydantic.Extra.forbid
+        allow_population_by_field_name = True
+
+    def _get_reader_method(self) -> str:
+        return self.type
+
+    def _get_reader_options_include(self) -> set[str] | None:
+        """These options are available as of spark v3.4.0
+
+        See https://spark.apache.org/docs/latest/sql-data-sources-parquet.html for more info.
+        """
+        return {
+            "datetimeRebaseMode",
+            "int96RebaseMode",
+            "mergeSchema"
+        }
```

Comment:
Should this instead use the snake case version e.g. `merge_schema`?

Reply:
No I don't think so, they should use the original spark names like in this PR. These options are passed on to spark directly via the batch_spec here: https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/execution_engine/sparkdf_execution_engine.py#L303